WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartMySitePrompts.kt

```diff
@@ -20,23 +20,23 @@ enum class QuickStartMySitePrompts constructor(
             R.id.my_site_scroll_view_root,
             R.id.row_view_site,
             R.string.quick_start_dialog_view_site_message_short,
-            R.drawable.ic_globe_grey_24dp,
+            R.drawable.ic_globe_white_24dp,
```

Comment:
Now that we rely on the tint to have the final color, I find it a little confusing to see the "white" in the drawable name. It feels minor but, I think I'd suggest to completely remove the color mention in the resource name so to avoid "misleading" the reader of the code here. The reader will have to hunt down the setting of the tint to understand what the color will be. WDYT?

Reply:
Hmm, but, is `ic_globe_white_24dp` ever going to be used as `white` indeed? My understanding is that we will always tint it (maybe via code it is as compat as possible) to some color and so, the "white" in its name will never actually apply. Is my assumption wrong?

Reply:
The two instances included in these changes, [`QuickStartMySitePrompts`](https://github.com/wordpress-mobile/WordPress-Android/pull/8881/commits/45bbc9f0ebc93676921b98523428a931459f79f0#diff-934fc2ed47696409f9d65e4bebe8aa1fR23) and [`my_site_fragment`](https://github.com/wordpress-mobile/WordPress-Android/pull/8881/commits/45bbc9f0ebc93676921b98523428a931459f79f0#diff-7c2f033014a455e52db8dd994c5f3e58R458), tint the icon from `grey` to `blue_light` and `grey_darken_30`, respectively.  There are two other instances, [`menu/reader_detail`](https://github.com/wordpress-mobile/WordPress-Android/blob/39efeded76007f21016edefc93db4fa0a5c77412/WordPress/src/main/res/menu/reader_detail.xml#L7) and [`menu/webview`](https://github.com/wordpress-mobile/WordPress-Android/blob/39efeded76007f21016edefc93db4fa0a5c77412/WordPress/src/main/res/menu/webview.xml#L7), which were using the `ic_globe_white_24dp` icon before these changes.

Ideally, we can reuse an existing drawable resource and delete an unnecessary drawable resource by replacing `ic_globe_grey_24dp` with `ic_globe_white_24dp`.  In the case of `ic_globe_grey_24dp`, it is used by the `wpIconDrawable` attribute in `WPLoginInputRow` views, which currently doesn't support tinting.  I tried to differentiate tinted and non-tinted icons with white and non-white naming, respectively.

Reply:
I agree.  I plan to write a post and update the ***Drawable Resources*** section in the `CONTRIBUTING.md` file to help other developers and our future selves with the tinting, naming, etc.

Reply:
Sounds good to me!